### DISCUSSION TITLE
Looking for developer/co-founder

### DISCUSSION THREAD

**Discussion by minhzc** [dreamworksanimation/openmoonray#88] [Ideas]
Hi friends.
I'm Ming, founder of TheStarium.com
I'm looking for a co-founder/MoonRay developer.
My company is working on a social network of virtual idols that is powered by AI and needs an automatic rendering application.
I see MoonRay could be the fittest option.
The project is launched, you can take a try at https://TheStarium.com
If anyone is interested in this, please inbox me for more details, or email: hello@thestarium.com

Thank you so much and best regards!

**Comment by ByteArchitect88** [comment]
Hi Ming,
I saw your post about the social network AI/MoonRay project.
I can work on the development based on your docs and goals.
Please share the full idea brief and roadmap.

aksoyfirat168@gmail.com
